amundsen_application/api/utils/metadata_utils.py

```diff
@@ -42,6 +42,8 @@ def marshall_table_full(table: Dict) -> Dict:
         'database',
         'is_view',
         'key',
+        # TODO - Consolidate with 'last_updated_epoch' to the same name and format.
+        'last_updated_timestamp',
```

Comment:
@jinhyukchang @feng-tao where do we stand with `epoch` vs `timestamp`? If I know what direction we're going, I can re-write FE code accept both formats and rename the field in Flask, then React can always refer to the correct name until this is renamed in the other services.

Reply:
Yeah, we settled with timestamp.

cc: @feng-tao